src/remote/inet.cpp

```diff
@@ -1844,8 +1845,23 @@ static int fork(SOCKET old_handle, USHORT flag)
 	start_crud.lpTitle = NULL;
 	start_crud.dwFlags = STARTF_FORCEOFFFEEDBACK;
 
+	//If stdout is file, then we configured to pass output from child
+	//so pass stdout/stderr handles to child process
+	HANDLE stdout_dup = INVALID_HANDLE_VALUE;
+	HANDLE stdout_handle = GetStdHandle(STD_OUTPUT_HANDLE);
+	if (GetFileType(stdout_handle) == FILE_TYPE_DISK)
+	{
+		if (DuplicateHandle(GetCurrentProcess(), GetStdHandle(STD_OUTPUT_HANDLE), GetCurrentProcess(),
+			&stdout_dup, 0, TRUE, DUPLICATE_SAME_ACCESS))
+		{
+			start_crud.hStdOutput = stdout_dup;
+			start_crud.hStdError = stdout_dup;
+		}
+		start_crud.dwFlags |= STARTF_USESTDHANDLES;
+	}
+
 	PROCESS_INFORMATION pi;
-	if (CreateProcess(NULL, cmdLine.begin(), NULL, NULL, FALSE,
+	if (CreateProcess(NULL, cmdLine.begin(), NULL, NULL, TRUE,
```

Comment:
We make child process to not inherit parent handles when fixed CORE-2356.
This patch undo that change.
Not sure it is good.